Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -4430,6 +4430,10 @@ - (void)tintColorDidChange
 
 - (BOOL)shouldAllowHighlightLinkCreation
 {
+    URL url { _page->currentURL() };
+    if (!url.isValid() || !url.protocolIsInHTTPFamily())
```

Comment:
it would theoretically be nice if this code and the below code could somehow have their logic shared between this and ContextMenuController, but that's probably not easy/trivial

Reply:
Agreed but they run in opposite processes :)